scripts/ffmprovisr

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env bash
 
 if [[ $OSTYPE = darwin* ]] ; then
+    default_browser=$(plutil -convert json ~/Library/Preferences/com.apple.LaunchServices/com.apple.launchservices.secure.plist -r -o - | grep https -b1 | tail -n1 | cut -d'"' -f4)
     if [ -d /usr/local/Cellar/ffmprovisr ] ; then
         ffmprovisr_path=$(find /usr/local/Cellar/ffmprovisr -iname 'index.html' | sort -M | tail -n1)
     fi
     if [ -z "${ffmprovisr_path}" ] ; then
         ffmprovisr_path='https://amiaopensource.github.io/ffmprovisr/'
     fi
-    open "${ffmprovisr_path}"
+    if [ -n "${default_browser}" ] ; then
```

Comment:
… that. So the `ffmprovisr_path` part and the `default_browser` part are grouped (easier to read and maintain).